onnxruntime/core/graph/model.cc

```diff
@@ -95,6 +95,10 @@ Model::Model(std::unique_ptr<ModelProto> model_proto, const IOnnxRuntimeOpSchema
         " specifies which version of the ONNX OperatorSet is being imported.");
   }
 
+  if (!model_proto->has_ir_version() || model_proto->ir_version() > ONNX_NAMESPACE::Version::IR_VERSION) {
```

Comment:
Conservatively, I think we should follow the same approach for the opset version. That is, we should explicitly define the max ir version in our source code, instead of using a value from ONNX (ONNX_NAMESPACE::Version::IR_VERSION)).

Reply:
@fdwr Can you take a look at this feedback?

Reply:
Sounds fine to me. We can define a constant in `model.h` `kMaximumIrVersion` per my original commit, before the PR feedback to use `ONNX_NAMESPACE::Version::IR_VERSION` instead: https://github.com/microsoft/onnxruntime/pull/2413/commits/ae4dc20ec250a79c63339a1258a651f7b3e3abe1

Reply:
This isn't a blocking issue. We can have the discussion later(with Pranav).